composer.json

```diff
@@ -12,7 +12,7 @@
     }
   ],
   "require": {
-    "php": ">=5.5",
+    "php": ">=7.0",
```

Comment:
that's a public bundle, is there a need for it ? some people might be using it with older version of php, we don't want to be unkind to those people :)